build-support/bin/cherry_pick.sh

```diff
@@ -8,6 +8,7 @@ set -e
 
 function fail {
   printf '%s\n' "$1" >&2
+  echo "Don't forget to remove the 'needs-cherrypick' label after making the PR!"
```

Comment:
Also minor, but only the very last call to `fail` (about merge conflicts) should have this extra message, as the earlier errors are before there's any hint of a PR to make.